manifests/12_prometheus_rule.yaml

```diff
@@ -65,3 +65,13 @@ spec:
           for: 15m
           labels:
             severity: warning
+    - name: custom-resource-detected
```

Comment:
I think this could be better named, since `custom-resource` is a generic term for an object in an api defined by a CRD.

`custom-source-detected` and `custom_source_total`?